USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -118,14 +189,65 @@ private string RenderMarker(Marker input)
                     output.AppendLine(mTMarker.Title);
                     output.AppendLine("</div>");
                     break;
+                case FMarker fMarker:
+                    foreach (Marker marker in input.Contents)
+                    {
+                        RenderMarker(marker);
+                    }
+                    break;
+                case FTMarker fTMarker:
+                    string footnote="";
+                    foreach(Marker marker in input.Contents)
+                    {
+                        switch (marker)

```

Comment:
This should be calling RenderMarker and not handling the rendering itself.

Reply:
Even though you're pushing all of these into list of strings you should still use the RenderMarker() function to generate the string contents that way it is consistent